01_P/P_1_0_01/sketch.js

```diff
@@ -18,24 +18,25 @@
 
 /**
  * changing colors and size by moving the mouse
- * 	 
+ *
  * MOUSE
  * position x          : size
  * position y          : color
- * 
+ *
  * KEYS
  * s                   : save png
  */
- 
+'use strict';
+
 function setup() {
   createCanvas(720, 720);
   noCursor();
+  colorMode(HSB, 360, 100, 100);
```

Comment:
Moved `colorMode`, `rectMode` and `noStroke` to the `setup` since they only have to be set once.